client/src/pages/ResetPassword/ResetPassword.tsx

```diff
@@ -0,0 +1,48 @@
+import { useState } from 'react';
+import { Box, Typography, Paper } from '@material-ui/core';
+import ResetPasswordForm from './ResetPasswordForm/ResetPasswordForm';
+import { useParams, Link } from 'react-router-dom';
+import { resetPassword } from '../../helpers/APICalls/forgotPassword';
+import useStyles from './useStyles';
+
+export default function Reset(): JSX.Element {
+  const [hasSubmitted, setHasSubmitted] = useState<boolean>(false);
+  const [error, setError] = useState<string>();
+  const classes = useStyles();
+  const { token, id } = useParams<{ id: string, token: string }>();
+
+  const handleSubmit = (
+    { password } : { password: string }
+  ) => {
+    resetPassword(password, token, id)
```

Comment:
If we encounter an error the first time, wouldn't we want to update the error state if the request is successful the next time around?